src/tests/unit/documents-serializer.test.js

```diff
@@ -0,0 +1,36 @@
+import chai from 'chai';
+import proxyquire from 'proxyquire';
+import { testMultipleResources, testSingleResource, createConfigStub } from './test-helpers';
+
+chai.should();
+
+describe('Test documents-serializer', () => {
+  const configStub = createConfigStub();
+  const { serializeDocuments, serializeDocument } = proxyquire(
+    'serializers/documents-serializer',
+    {},
+  );
+  configStub.restore();
+
+  it('serializeDocuments should properly serialize multiple documents in JSON API format', () => {
+    const doc = {
+      typeId: 'fake id',
+    };
+    const result = serializeDocuments([doc, doc], '');
```

Comment:
Also a minor stuff, since all the other fake stuff you named "fake something", should this be named in the same way?